regolith/helpers/a_projectumhelper.py

```diff
@@ -366,3 +371,30 @@ def insert_checklists(self, pdoc, now):
             "status": "finished"
         }})
         return pdoc
+
+    # create Google Doc page for prum in Google Drive 'bg-projects'
+    def gdoc_generator(self):
+        # To use prum name for generated document name
+        rc = self.rc
+
+        # Authenticate user
+        store = oauth2client.file.Storage('credentials.json')
+        credentials = store.get()
+        http = credentials.authorize(Http())
+        drive = discovery.build('drive', 'v3', http=http)
+
+        # Use Gdoc API to generate blank document in 'bg-projects'
+        folder_id = '12yfN_X0k9NiLDt9pe14JhLyqULxM2-vK'
```

Comment:
group specific information should not be hard-coded but should be pulled from regolithrc.json (please discuss with Kayla if you are not sure)